meilisearch-lib/src/tasks/error.rs

```diff
@@ -14,6 +14,8 @@ pub enum TaskError {
     UnexistingTask(TaskId),
     #[error("Internal error: {0}")]
     Internal(Box<dyn std::error::Error + Send + Sync + 'static>),
+    #[error("Task id is out of bounds.")]
+    OutOfBounds(),
```

Comment:
no parenthesis is necessary here
```suggestion
    OutOfBounds,
```